config/core/test-configs.yaml

```diff
@@ -155,6 +155,17 @@ test_plans:
       - passlist: &qemu_labs_filter
           lab: ['lab-baylibre', 'lab-collabora-staging']
 
+  chromeos-flash:
+    pattern: 'chromeos/chromeos-flash-template.jinja2'
+    params:
+      job_timeout: '15'
+      docker_image: 'kernelci/chromeos-flash'
+      chromeos_image: 'chromiumos_test_image.bin'
```

Comment:
As this is an `octopus` image, we probably should include `octopus` in the file name or in a sub-directory, ideally with the Chrome OS release number as well, to better differentiate with future images.  The `chromiumos_test_image.bin` name is the standard one produced by the Chrome OS SDK build system.

Reply:
I see you added the Chrome OS release number to the file name but not `octopus`. Is that on purpose? I believe the image was built specifically for `octopus` but we could double check with Brian.